docs/src/pages/components/tables/CustomPaginationActionsTable.js

```diff
@@ -152,7 +152,7 @@ export default function CustomPaginationActionsTable() {
           <TableFooter>
             <TableRow>
               <TablePagination
-                rowsPerPageOptions={[5, 10, 25]}
+                rowsPerPageOptions={[5, 10, 25, -1]}
```

Comment:
What do you think of this API?
```suggestion
                rowsPerPageOptions={[5, 10, 25, { value: -1, label: 'All}]}
```

This enables developers to translate the option.

Reply:
This is definitely much better. I have amended the API.